core/src/scss/components/local-footer/_local-footer.scss

```diff
@@ -0,0 +1,310 @@
+.su-local-footer__cell2 {
+  grid-area: B;
+  vertical-align: top;
+
+  @include grid-media-between('xs', 'md') {
+    @include modular-spacing('padding-bottom', 1);
+  }
+
+  @include grid-media-only('md') {
+    grid-area: C;
+  }
+
+  // Handle the list of navs with a bit of padding and sizing
+  // to get faux columns.
+  nav {
+    vertical-align: top;
+
+    @include grid-media('xl') {
+      $_gap: map-get($su-grid-gutters, 'xl') / 2;
+      min-width: calc(49% - #{$_gap});
+      max-width: calc(49% - #{$_gap});
+      display: inline-block;
+
+      &:last-of-type {
+        @include margin(null null null $_gap);
+      }
+
+      &:first-of-type {
+        @include margin(null $_gap null null);
+      }
+    }
+
+    @include grid-media('2xl') {
+      $_gap: map-get($su-grid-gutters, '2xl') / 2;
+
+      &:last-of-type {
+        @include margin(null null null $_gap);
+      }
+
+      &:first-of-type {
+        @include margin(null $_gap null null);
+      }
+    }
+
+    &:first-of-type {
+      @include modular-spacing('padding-bottom', 1);
+    }
+  }
+}
+
+// Specific to cell3 styles and structure.
+// ------------------------------------------------
+.su-local-footer__cell3 {
+  grid-area: C;
+
+  @include grid-media-only('md') {
+    grid-area: B;
+  }
+}
+
+// List heading element styles.
+// ------------------------------------------------
+.su-local-footer__list-heading {
+  @include type-e;
+}
+
+// Common styles for address and links.
+// ------------------------------------------------
+.su-local-footer__address,
+.su-local-footer__action-links {
+  @include modular-spacing('padding-bottom', 3);
+}
+
+// Address tag formatting and styles.
+// ------------------------------------------------
+.su-local-footer__address {
+  line-height: 140%;
+}
+
+// Styling for the action links section.
+// ------------------------------------------------
+.su-local-footer__action-links {
+  a {
+    @include link-icon(arrow-right, 0.8em, right, -0.18em);
+    font-weight: $su-font-semi-bold;
+  }
+}
+
+// Styles for the social links.
+// ------------------------------------------------
+.su-local-footer__social-links {
+  @include list-horizontal;
+
+  li {
+    @include padding(null 18px null null);
+
+    &:last-child {
+      @include padding(null 0 null null);
+    }
+  }
+
+  i {
+    font-size: 2.5rem;
+
+    // FA is included as a vendor dependency but the class styles are not
+    // included so we have to use the mixins and font declarations here.
+    &::before {
+      @include fa-icon;
+      color: $su-color-black;
+      font-family: 'Font Awesome 5 Brands';
+      transition: color 0.25s ease-out;
+    }
+  }
+
+  a {
+    span {
+      @include sr-only;
+    }
+  }
+
+  // Font Awesome icons for specific selectors.
+  // ------------------------------------------------
+  .su-local-footer__social-facebook {
+    &::before {
+      content: fa-content($fa-var-facebook-f);
+    }
+
+    &:hover {
```

Comment:
```suggestion
    &:hover,
    &:focus {
```

Reply:
Could we add the color change for on focus also (and for all the other social icons please)?